index.html

```diff
@@ -2,13 +2,252 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="icon" href="public/icon-list.png" alt="list-icon" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Josefin+Sans:ital@1&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="src/css/style.css" />
     <title>Abdullahi Todo List App</title>
   </head>
 
-  <body class="m-0 flex min-h-screen min-w-80 place-items-center">
-    <main class="mx-auto my-0 max-w-7xl p-8 text-center" id="app" role="main"></main>
-    <script src="src/js/main.js" type="module"></script>
+  <body class="bg-blue-200 text-lg">
+    <main role="main">
+      <section class="relative m-auto flex w-96 flex-col justify-center">
+        <div
+          class="fixed left-0 right-0 top-0 -z-10 h-1/3 w-screen bg-mobile-light bg-cover bg-no-repeat"
+        ></div>
+        <div class="my-28 mb-8 flex size-full justify-between">
+          <h1 class="text-4xl font-bold not-italic tracking-widest text-blue-100">TODO</h1>
+          <img class="h-8 w-8 cursor-pointer" src="images/icon-moon.svg" alt="theme-icon" />
```

Comment:
We can use `size-` shorthand for `h-8 w-8`

Reply:
Done